time_execution/backends/elasticsearch.py

```diff
@@ -89,9 +99,12 @@ def write(self, name, **data):
         data["name"] = name
         data["timestamp"] = datetime.utcnow()
 
-        self.client.index(
-            index=self.get_index(),
-            doc_type=self.doc_type,
-            id=None,
-            body=data
-        )
+        try:
```

Comment:
This looks good to avoid the api from breaking because of potential failures on the network connection to elastic. But what about the performance impact of the TCP connection overhead?